plugins/rtmp-services/data/locale/en-US.ini

```diff
@@ -8,3 +8,5 @@ UseAuth="Use authentication"
 Username="Username"
 Password="Password"
 ShowAll="Show all services"
+MultitrackVideo.TwitchDisclaimer="Enhanced Broadcasting automatically optimizes your settings to encode and send multiple video qualities to Twitch, powering the best possible viewer experience. Selecting this option will send Twitch information about your computer and software setup. <a href='https://help.twitch.tv/s/article/multiple-encodes'>Learn More</a>"
+MultitrackVideo.Disclaimer="%1 automatically optimizes your settings to encode and send multiple video qualities to %2, powering the best possible viewer experience. Selecting this option will send %2 information about your computer and software setup."
```

Comment:
I'm wondering if we need two distinct strings here or if we could just parameterize one (already done with `MultitrackVideo.Disclaimer`) and add a second parameterized string for the "Learn More" link that can be optionally configured and appended to the first string before displaying it. That would avoid hard-coding a nearly identical service-specific string here.

Reply:
changed this in https://github.com/obsproject/obs-studio/pull/10543/commits/72568fcdb031a14deee0fe45a678844d9e9bd26f (<https://github.com/obsproject/obs-studio/pull/10543#event-12615445003>)

Reply:
Thanks. Resolving.